src/common/stmt-syntax.k

```diff
@@ -25,12 +26,13 @@ syntax StmtWithoutTrailing ::= Block
 |SynchronizedStmt
 
 syntax BlockStmt ::= Stmt|LocalVarDecStmt|ClassDec
-syntax LocalVarDecStmt ::= Type Id ";" [klabel('LocalVarDec)]
-syntax ClassBody ::= "{" KListWrap "}" [klabel('ClassBody)]
-syntax ClassDecHead ::= KListWrap "class" Id K K K [klabel('ClassDecHead)]
+syntax LocalVarDecStmt ::= LocalVarDec ";" [klabel('LocalVarDecStm)]
+syntax LocalVarDec ::= AnnoVarModList Type VarDecList [klabel('LocalVarDec)]
+syntax ClassBody ::= "{" ClassBodyDecList "}" [klabel('ClassBody)]
+syntax ClassDecHead ::= AnnoClassModList "class" Id K K K [klabel('ClassDecHead)]
 syntax ClassDec ::= ClassDecHead ClassBody [klabel('ClassDec)]
 
-syntax Block ::= "{" BlockStmt "}" [klabel('Block)]
+syntax Block ::= "{" K "}" [klabel('Block)]
```

Comment:
We'd like to avoid using the K sorts (K, KItem, KList, etc.) in definitions, because they require more knowledge about K in order to understand them and can lead to subtle errors.  Why did you have to use K instead of BlockStmt above?